### PR TITLE
Better keyvaluepage align in reading statistics

### PR DESCRIPTION
**Pull request by robert00s**
Close: #3515 

Now it looks like this:

koreader_029

### CODE REVIEW

File: frontend/apps/filemanager/filemanagerbookinfo.lua [hunk 1]
```diff
@@ -181,7 +181,6 @@ function BookInfo:show(file, book_props)
 
     local widget = KeyValuePage:new{
         title = _("Book information"),
-        value_overflow_align = "right",
```
Comment (poire-z) [inline frontend/apps/filemanager/filemanagerbookinfo.lua]: Why remove it from here ? (it was done in #3407).
Here, we would prefer `value_overflow_align = "right" `and `value_align = "left"`.

File: frontend/ui/widget/keyvaluepage.lua [hunk 2]
```diff
@@ -184,6 +184,9 @@ function KeyValueItem:init()
             self.show_value = self.value
         end
     else
+        if self.value_overflow_align == "right" then
+            key_w = frame_internal_width - value_w_rendered
+        end
```
Comment (poire-z) [inline frontend/ui/widget/keyvaluepage.lua]: I think you should have another setting named simply `value_align`, that applies if overflow or not.
(Here, you put `if value_overflow_align` in the `else` section dealing with case when there is no overflow - key and value both fit in their allowed 50%.)
So, I guess you would have here: `value_overflow_align = "right"` and `value_align = "right"`

**Comment by Frenzie** [comment]
Btw, what was the workaround again to some early Stats v2 issue where you don't get any stats on books? Just delete the sqlite file? 

(I don't really use the stats but I just noticed that I supposedly haven't read anything… I'm pretty sure it used to count something in the past. :-P)

File: frontend/ui/widget/keyvaluepage.lua (hunk 2 above)
Comment (robert00s) [reply frontend/ui/widget/keyvaluepage.lua]: This is a much better solution.

**Comment by robert00s** [comment]
@Frenzie 
This should help:
https://github.com/koreader/koreader/issues/3375#issuecomment-338116652

**Comment by poire-z** [comment]
Looking better indeed :)

**Comment by Frenzie** [comment]
@robert00s Thanks. I've now already read more pages (3) than the thousands (counted as 2) from the past months. :-P

**Pull request merged**